src/components/localization/localized-link.js

```diff
@@ -164,7 +164,7 @@ const getURLFormat = (type, locale, to, affiliate_lang) => {
             type.replace(/_/g, '-') + '/'
         }`
     } else if (only_en_new_tab_no_modal.includes(type)) {
-        return `${localized_link_url.domain_full_url}/${type.replace(/_/g, '-') + '/'}`
```

Comment:
I wonder, if this would break other things or not

Reply:
@habib-deriv,  prior to updating my master branch, the slash was missing but nothing seemed broken though
but, I'll try & test all other pages

Reply:
It should't break. As I've tested. But Fiona's test will necessary too, I think

Reply:
These are for pages outside of this project right?